test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/QueueViewSpec.scala

```diff
@@ -17,7 +17,7 @@
 package uk.gov.hmrc.tariffclassificationfrontend.views
 
 import uk.gov.hmrc.tariffclassificationfrontend.models.{Case, Paged, Queue}
-import uk.gov.hmrc.tariffclassificationfrontend.views.ViewMatchers._
+import uk.gov.hmrc.tariffclassificationfrontend.views.ViewMatchers.{containElementWithID, _}
```

Comment:
`containElementWithID` can be removed here.
In fact, `_` includes it already :)